packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -259,6 +262,37 @@ export function TransferPanelSummary({ token }: TransferPanelSummaryProps) {
           )}
         </span>
       </div>
+      {(isDestinationChainArbitrumOne || isDestinationChainArbitrumSepolia) && (
```

Comment:
we need to make sure it's withdrawal mode
because otherwise this element will also show for deposits to Arb One or Arb Sepolia

e.g.
<img width="441" alt="image" src="https://github.com/user-attachments/assets/b7b8eef4-c300-45b0-8d12-2c7cbc78e683">